src/android/lib/LaunchNavigator.java

```diff
@@ -89,6 +90,7 @@
         _supportedAppPackages.put(WAZE, "com.waze");
         _supportedAppPackages.put(YANDEX, "ru.yandex.yandexnavi");
         _supportedAppPackages.put(SYGIC, "com.sygic.aura");
+        _supportedAppPackages.put(SYGIC_NAVIGATION, "com.sygic.aura");
```

Comment:
`SYGIC_NAVIGATION`has the same app package name as `SYGIC`: `com.sygic.aura` 
how is it any different from `SYGIC`?

Reply:
Unfortunately thats true, i will cancel this PR. This change can only be used in my case.